src/main/java/org/digma/intellij/plugin/insights/AbstractInsightsMessageRouterHandler.java

```diff
@@ -99,6 +99,7 @@ private void pushIssuesListData(JsonNode jsonNode) throws JsonProcessingExceptio
                 query.at("/sorting/criterion").asText(null),
                 query.at("/sorting/order").asText(null),
                 query.at("/insightTypes").toString(),
+                query.at("/services").toString(),
```

Comment:
maybe its necessary to check isMissingNode before calling toString? I'm not sure

Reply:
`at()` will return missing node if it not found an `toString` produce emptyString